libmscore/score.cpp

```diff
@@ -3584,36 +3584,41 @@ ChordRest* Score::findCR(int tick, int track) const
 
 //---------------------------------------------------------
 //   findCRinStaff
-//    find chord/rest <= tick in staff
-//    prefer shortest duration (so it does not overlap next chordrest segment)
+//    find last chord/rest on staff that ends before tick
 //---------------------------------------------------------
 
 ChordRest* Score::findCRinStaff(int tick, int track) const
```

Comment:
It's a detail but since this method is called `findCRin*Staff*` maybe it would be clearer to pass a staffIdx and not a track.